language/ruby/language.go

```diff
@@ -62,9 +66,35 @@ func (l *Language) ExecuteTests(logger *log.Logger, repositoryPath string) (test
 	return testResult, problems, nil
 }
 
+var mistakesRe = regexp.MustCompile(`\d\) Error:\n\w+#\w+:\n(.*)\n\s*(.*)`)
+
 // Mistakes builds a Ruby repository and returns the list of mistakes found.
 func (l *Language) Mistakes(logger *log.Logger, repositoryPath string) (mistakes []string, err error) {
-	logger.Panic("not implemented")
+	output, err := util.CommandWithResult(context.Background(), logger, &util.Command{
+		Command: []string{
+			"rake",
+			"test",
```

Comment:
Please document that this is sub-optimal since it only works if there are tests for sure. I.e. we might introduce different ruby tasks in the future where just a script is run. So this won't work then anymore.